crates/sncast/src/starknet_commands/script/run.rs

```diff
@@ -143,6 +143,7 @@ impl<'a> ExtensionLogic for CastScriptExtension<'a> {
                     WaitForTx {
                         wait: true,
                         wait_params: self.config.wait_params,
+                        silent: false,
```

Comment:
nit: `silent` seems too general, maybe `suppress_ui`?

Reply:
Renamed to `show_ui_outputs`